cmd/juju/commands/exec.go

```diff
@@ -246,8 +250,8 @@ func ConvertActionResults(result params.ActionResult, query actionQuery) map[str
 	return values
 }
 
-func (c *runCommand) Run(ctx *cmd.Context) error {
-	client, err := getRunAPIClient(c)
+func (c *execCommand) Run(ctx *cmd.Context) error {
```

Comment:
Doesn't look possible with juju/cmd, but ideally we should be able to warn when a deprecated alias is used. Something like "WARN: juju exec will soon supersede juju run".

Reply:
Yeah, not really possible. We'll highlight in the juju 3 release notes